modules/serverless-kafka-broker-with-isolated-dataplane.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * serverless/eventing/brokers/serverless-using-brokers.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-broker-with-isolated-dataplane_{context}"]
+= Creating an Apache Kafka broker that uses an isolated data plane
```

Comment:
```suggestion
= Creating a Knative broker for Apache Kafka that uses an isolated data plane
```

Reply:
I have a separate JIRA for renaming to "Knative broker for Apache Kafka" but I will update this occurrence in this JIRA when I add the TP warning